src/mir/index.md

```diff
@@ -255,7 +255,66 @@ but [you can read about those below](#promoted)).
 
 ## Representing constants
 
-*to be written*
+When code has reached the MIR stage, constants can generally come in two forms: *MIR constants* ([`mir::Constant`]) and *type system constants* ([`ty::Const`]).
```

Comment:
@oli-obk this makes me wonder if we should rename `mir::Constant` to `mir::Const` to match `ty::Const`...

Reply:
yea :laughing: I wanted this forever, but never did it

Reply:
I can make a PR, but let's land https://github.com/rust-lang/rust/pull/115865 first or this will be a hell of a conflict.^^